source/cloud-security/amazon/services/supported-services/security-hub.rst

```diff
@@ -0,0 +1,176 @@
+.. Copyright (C) 2015, Wazuh, Inc.
+
+.. meta::
+   :description: Learn how to configure Amazon Security Hub findings and insights fetching.
+
+AWS Security Hub
+================
+
+.. versionadded:: 4.9.0
+
+`AWS Security Hub <https://aws.amazon.com/security-hub/>`_ is a cloud security posture management (CSPM) service that automates security best practice checks, aggregates security alerts into a determined format, and helps the user understand the overall security posture across all of the AWS accounts.
+
+Security Hub helps users assess their compliance against security best practices as follows:
+
+-  Runs checks against security controls.
+-  Generates control `findings <https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-findings.html>`__.
+-  Groups related findings into collections called `insights <https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-insights.html>`__.
+
+Wazuh integrates with `Amazon SQS <https://aws.amazon.com/sqs>`_ and `EventBridge <https://aws.amazon.com/eventbridge>`_ to centralize Security Hub findings and insights in a single place. To set up the integration, you need to:
+
+   #. Configure AWS:
```

Comment:
We can use three-spaces indentation here as follows:

```
#. Configure AWS:

   -  Enable Amazon Security Hub.
   -  Enable an Amazon SQS queue.
   -  Enable an Amazon S3 bucket with Event notifications since for every Security Hub object creation event, the bucket sends notifications to the queue.
#. Set up the Wazuh integration for Amazon Security Hub.
```

replacing the current 

```
   #. Configure AWS:

      - Enable Amazon Security Hub.
      - Enable an Amazon SQS queue
      - Enable an Amazon S3 bucket with Event notifications since for every Security Hub object creation event, the bucket sends notifications to the queue.
   #. Set up the Wazuh integration for Amazon Security Hub.
```